web/src/features/panels/zone/FeedbackCard.tsx

```diff
@@ -0,0 +1,259 @@
+import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group';
+import Pill from 'components/Pill';
+import { Dispatch, SetStateAction, useState } from 'react';
+import { HiOutlineX } from 'react-icons/hi';
+import { useTranslation } from 'translation/translation';
+
+export default function FeedbackCard({
+  estimationMethod,
+}: {
+  estimationMethod?: string;
+}) {
+  const [isClosed, setIsClosed] = useState(false);
+  const [state, setState] = useState('1');
+
+  const handleClose = () => {
+    setIsClosed(true);
+  };
+
+  const title = getQuestionTranslation('title', state);
+  const subtitle = getQuestionTranslation('subtitle', state);
+
+  if (isClosed) {
+    return null;
+  }
+
+  return (
+    <div
+      data-test-id="feedback-card"
+      className="mb-4 flex w-full flex-col rounded-lg border border-neutral-200 bg-zinc-50 pl-2.5 transition-all dark:border-gray-700 dark:bg-gray-900"
+    >
+      <div className="flex flex-row items-center justify-between">
+        <div className="flex flex-initial flex-row gap-2">
+          <div
+            className={`h-[16px] w-[16px] bg-[url('/images/emaps-icon-light.svg')] bg-contain bg-center dark:bg-[url('/images/emaps-icon-dark.svg')]`}
+          />
+          <h2
+            className={`self-center text-left text-sm font-semibold text-black dark:text-white`}
+            data-test-id="title"
+          >
+            {title}
+          </h2>
+        </div>
+        <button data-test-id="close-button" onClick={handleClose} className="px-3 py-2.5">
+          <HiOutlineX />
+        </button>
+      </div>
+      <div className="pb-2 pr-2.5">
+        <div
+          className="pb-1 text-xs font-medium text-neutral-400"
+          data-test-id="subtitle"
+        >
+          {subtitle}
+        </div>
+        <FeedbackActions
+          state={state}
+          setState={setState}
+          estimationMethod={estimationMethod}
+        />
+      </div>
+    </div>
+  );
+}
+
+function InputField({
```

Comment:
It would be useful if the input could grow with the users input, you could use a [textarea element](https://react.dev/reference/react-dom/components/textarea) here.

Reply:
I changed it in the latest commit. Does it look like you envisioned?

Reply:
Looks great!